Automattic/sensei
# Implement question import task

### Overview
This PR implements the question import task.

- Every field should be imported correctly.
- Question are compared by the slug to check if it is an existing course or a new one. If the question exists already, it is updated.
- 

### Testing instructions
- To upload a CSV file you can navigate to Sensei LMS > Import. Once you upload the file click continue to start the import job.
- Make sure that the cronjobs have run.
- Alternatively you can run the job by using the CLI command `wp --user=<admin> --questions=<csv file> sensei-import`.
- Observe the imported questions.
- For an example CSV file you can see in the attached files.

[question-sample copy.csv.txt](https://github.com/Automattic/sensei/files/4783157/question-sample.copy.csv.txt)
